examples/repl.rs

```diff
@@ -58,6 +64,25 @@ mod example {
         disassembler: DisassemblySource,
     }
 
+    /// The subcommands that are acceptable for backtrace.
+    enum BacktraceType {
```

Comment:
Please implement `HighlightAndComplete` for `BacktraceType` and at line 46 (`Backtrace|bt: String`) instead of `String`.

Reply:
@bjorn3 Can you elaborate more on this ?

Reply:
@bjorn3 Please pardon my pure rust skills. Let me be explicit and ask, you want the line 46 to change from

```rust
Backtrace|bt: String,
```
to 

```rust
Backtrace|bt: (Backtrace|bt: String),
```

Reply:
No, change it to `Backtrace|bt: BacktraceType,`. That requires implementing `HighlightAndComplete` for `BacktraceType`.

Reply:
done!